open-api.yaml

```diff
@@ -748,6 +748,16 @@ components:
 
         > warn
         > Attribute(s) used in `filter` should be declared as filterable attributes. See [Filtering and Faceted Search](https://docs.meilisearch.com/reference/features/filtering_and_faceted_search.html).
+    fields:
+      name: fields
+      in: query
+      description: 'Comma-separated list of fields to display for an API resource. By default it contains all fields of an API resource.'
```

Comment:
The fields list is case-sensitive?

Reply:
That's a good question, and I think it should ease the usage.

@Kerollmops It is already the case natively? I know that query parameter are not so easy to handle due to some limitations.